scenario_player/node_support.py

```diff
@@ -332,7 +336,11 @@ def _command(self):
             if option_name in self._options:
                 cmd.append(f"--{option_name}")
                 if option_name not in FLAG_OPTIONS:
-                    cmd.append(self._options[option_name])
+                    option_value = self._options[option_name]
+                    if isinstance(option_value, list):
```

Comment:
This doesn't look right. The `--{option_name}` already get's appended in line 337.